<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="load">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>20</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>430</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="learn">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>430</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="condition">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>500</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>學習率</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>500</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>收斂條件</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>測試辨識率</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>訓練辨識率</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="menu1">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maxCount">
     <number>2147483644</number>
    </property>
    <item>
     <property name="text">
      <string>perceptron1.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2Ccircle1.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2Circle1.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2Circle2.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2CloseS.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2CloseS2.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2CloseS3.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2cring.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2CS.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2Hcircle1.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2ring.txt</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>perceptron2.txt</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>491</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="ltest"/>
   </widget>
   <widget class="QPushButton" name="clear_layout">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>70</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="menu2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>標楷體</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>隱藏層層數</string>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>300</y>
      <width>241</width>
      <height>101</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>神經元個數</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>1</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
